src/webgpu/util/math.ts

```diff
@@ -2056,6 +2056,7 @@ export function isPowerOfTwo(n: number): boolean {
   if (!Number.isInteger(n)) {
     return false;
   }
+  assert((n | 0) === n, 'isPowerOfTwo only supports 32-bit numbers');
```

Comment:
I verified that the tests that use this helper still pass.
Could switch to `Math.log2(n) % 1 === 0` but I'm not 100% if that's a sound implementation either so I just did this.
`webgpu:api,validation,capability_checks,limits,*`
`webgpu:api,operation,adapter,requestDevice:*`